src/detect-content.c

```diff
@@ -2309,6 +2312,35 @@ int DetectContentParseTest44(void)
     return result;
 }
 
+/**
+ * \test Parsing test
```

Comment:
can you add a comment saying that it's about the missing ; after the first content?